cumulus/polkadot-omni-node/lib/src/cli.rs

```diff
@@ -209,6 +222,16 @@ pub struct Cli<Config: CliConfig> {
 	pub(crate) _phantom: PhantomData<Config>,
 }
 
+/// Development sealing mode.
+#[derive(Debug, Clone, Copy)]
+pub enum DevSealMode {
```

Comment:
does this need to be pub? check-semver fails because it detects this new pub type as a public interface change. Maybe this will help:
```suggestion
pub(crate) enum DevSealMode {
```